mob

```diff
@@ -440,12 +441,19 @@ docker_run.extend([tag])
 
 docker_run.extend(["/bin/bash"])
 
+if args.cmd is not None:
+    docker_run.extend(["-c"])
+    docker_run.extend([" ".join(args.cmd)])
+
 try:
     maybe_run(docker_run)
 except subprocess.CalledProcessError as exception:
     if args.action == 'prompt' and exception.returncode == 130:
         # This is normal exit of prompt
         sys.exit(0)
+    if args.action == 'prompt' and args.cmd is not None:
+        # Make sure custom commands have their expected return codes
+        sys.exit(exception.returncode)
```

Comment:
IIUC we should check for `cmd` first, so we don't swallow an exit code of 130 as normal.

```suggestion
    if args.action == 'prompt' and args.cmd is not None:
        # Make sure custom commands have their expected return codes
        sys.exit(exception.returncode)
    if args.action == 'prompt' and exception.returncode == 130:
        # This is a normal exit of prompt
        sys.exit(0)
```